Source/ACE.Server/Command/Handlers/AdminCommands.cs

```diff
@@ -1865,6 +1866,111 @@ public static void HandleMagicGod(Session session, params string[] parameters)
             ChatPacket.SendServerMessage(session, "You are now a magic god!!!", ChatMessageType.Broadcast);
         }
 
+        
+        [CommandHandler("modifyvital", AccessLevel.Admin, CommandHandlerFlag.None, 2, "Adjusts the maximum vital attribute for the last appraised mob/player and restores full vitals", "<Health|Stamina|Mana> <delta>")]
+        public static void HandleModifyVital(Session session, params string[] parameters)
+        {
+            var lastAppraised = CommandHandlerHelper.GetLastAppraisedObject(session);
+            if (lastAppraised == null || !(lastAppraised is Creature)) return;
```

Comment:
Might be useful to output text to the admin letting them know why the command failed.